demisto_sdk/commands/validate/tests/validators_test.py

```diff
@@ -433,9 +437,20 @@ def test_script_valid_rn(self, mocker, file_path, file_type):
         file_validator.validate_added_files(file_path, file_type)
         assert file_validator._is_valid
 
+    def test_pack_validation(self):
+        file_validator = FilesValidator(validate_conf_json=False)
+        file_validator.file_path = VALID_PACK
+        file_validator.is_valid_structure()
+        assert file_validator._is_valid is False
+
+    ARE_TEST_CONFIGURED_TEST_INPUT = [
+        (VALID_INTEGRATION_TEST_PATH, True),
+        (INVALID_INTEGRATION_NO_TESTS, False),
+        (INVALID_INTEGRATION_NON_CONFIGURED_TESTS, False)
+    ]
 
-def test_pack_validation():
-    file_validator = FilesValidator(validate_conf_json=False)
-    file_validator.file_path = VALID_PACK
-    file_validator.is_valid_structure()
-    assert file_validator._is_valid is False
+    @pytest.mark.parametrize('file_path, expected', ARE_TEST_CONFIGURED_TEST_INPUT)
+    def test_are_tests_configured(self, file_path, expected):
```

Comment:
Please document the unit test as described [here](https://panw-global.slack.com/archives/G011E63JXPB/p1587370983267700).